front/app/containers/Admin/projects/project/analysis/hooks/useAnalysisFilterParams.ts

```diff
@@ -21,16 +21,34 @@ const useAnalysisFilterParams = () => {
     'comments_to',
   ]);
 
+  const tag_ids = handleArraySearchParam(searchParams, 'tag_ids');
+
   const arrayFilters = {
-    tag_ids: handleArraySearchParam(searchParams, 'tag_ids'),
+    ...(tag_ids
+      ? { tag_ids: handleArraySearchParam(searchParams, 'tag_ids') }
+      : {}),
   };
 
+  const authorCustomArrayFilters = {};
```

Comment:
This works! Generally, we prefer a more functional approach to deriving values in React components (in this case reduce could be useful). Fine to keep it as is, as well. Maybe it would be a bit better put it in a method that returns `authorCustomArrayFilters` and `authorCustomScalarFilters` instead of having those as top level consts in the component.

Reply:
:+1: Done